pkg/apis/volumesnapshot/v1beta1/types.go

```diff
@@ -31,6 +31,12 @@ import (
 // +kubebuilder:object:root=true
 // +kubebuilder:resource:scope=Namespaced
 // +kubebuilder:subresource:status
+// +kubebuilder:printcolumn:name="Ready",type=boolean,JSONPath=`.status.readyToUse`,description="Indicates if a snapshot is ready to be used to restore a volume."
+// +kubebuilder:printcolumn:name="Source",type=string,JSONPath=`.spec.source.persistentVolumeClaimName`,description="Specifies where a snapshot will be created from."
```

Comment:
For pre-provisioned snapshots, the source is volumeSnapshotContentName.  How do we show that?

Reply:
I don't think there's a way that we can toggle between the two. As of now I've added both sources.